torch/csrc/monitor/python_init.cpp

```diff
@@ -137,17 +244,35 @@ void initMonitorBindings(PyObject* module) {
   py::implicitly_convertible<bool, data_value_t>();
 
   py::class_<PythonEventHandler, std::shared_ptr<PythonEventHandler>>
-      eventHandlerClass(m, "PythonEventHandler");
-  m.def("register_event_handler", [](std::function<void(const Event&)> f) {
-    auto handler = std::make_shared<PythonEventHandler>(f);
-    registerEventHandler(handler);
-    return handler;
-  });
+      eventHandlerClass(m, "EventHandler", R"DOC(
+        EventHandler is a wrapper type returned by ``register_event_handler``
```

Comment:
I missed this earlier, but this may be confusing, since we register event_handlers with a `handler`. Maybe EventHandlerIdentifier?

Reply:
renamed